Makefile

```
@@ -24,36 +27,79 @@
-	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  $(BLUE)%-15s$(NC) %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  $(BLUE)%-20s$(NC) %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
 ## Development
 build: ## Build the application
 	@echo "$(GREEN)Building $(APP_NAME)...$(NC)"
 	go build -o bin/$(APP_NAME) cmd/server/main.go
 	@echo "$(GREEN)Build complete!$(NC)"
 
-test: ## Run tests
-	@echo "$(GREEN)Running tests...$(NC)"
+test: ## Run tests across all languages
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (13).

(maxbodylength)